.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+# Settings set up https://pre-commit.com/#pre-commit-configyaml---top-level
```

Comment:
What do we need this file for?

Reply:
This file is configuration for pre-commit framework. It has hook settings.

Reply:
But we don't use any framework here, we just copy a bash-script into designated place according to `git hooks` manual.

Reply:
Yes, you're right. I removed pre-commit framework from readme and requirements now.